scripts/data_files/driver_templates/psa_crypto_driver_wrappers.h.jinja

```diff
@@ -2761,6 +2752,11 @@ static inline psa_status_t psa_driver_wrapper_pake_output(
     size_t output_size,
     size_t *output_length )
 {
+    (void) step;
+    (void) output;
+    (void) output_size;
+    (void) output_length;
+
```

Comment:
Those look useful, but it also seems to be they were perfectly reachable in their previous location - otherwise the `return` statement after `default:` would be unreachable as well.